5_databases/week3/Meal.sql

```diff
@@ -0,0 +1,31 @@
+use homework3;
+--1 Get all meals--
+SELECT*
+from `Meal`;
+--2 ADD new meal --
+INSERT into Meal (`id`,`title`,`description`,`location`,`when`,`max_reservations`,`price`,`created_date`)
+VALUES(1,'couscous','spicy','Tunisia','2023-09-10','10','60','2023-09-20');
+INSERT into Meal (`id`,`title`,`description`,`location`,`when`,`max_reservations`,`price`,`created_date`)
+VALUES(2,'sushi','crispy','japon','2023-04-10','30','50','2023-09-10');
+INSERT into Meal (`id`,`title`,`description`,`location`,`when`,`max_reservations`,`price`,`created_date`)
+VALUES(3,'pizza','tomato','italy','2023-02-10','3','20','2023-09-20');
+INSERT into Meal (`id`,`title`,`description`,`location`,`when`,`max_reservations`,`price`,`created_date`)
+VALUES(4,'Rød grød med','meduim','Denmark','2023-03-10','3','20','2023-09-10');
+--3 Get a meal with any id--
+SELECT*
+from `Meal`
+WHERE id = 1;
+
+--4 update a meal any id --
+UPDATE `Meal`
+SET id = 2
+WHERE id = 1;
```

Comment:
You generally never want to update a primary key of a row, i.e. `id`. The primary key should be set when the row is inserted in the database and then never changed. `id` is only used internally to make each row uniquely identifiable.

Reply:
I also think this was never a task. 😄